apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -386,20 +403,22 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupInfered
                 selectedCustomInput={selectedCustomInput}
                 onSubmit={(values) => {
                   const customInput: CustomInputParsed = {
-                    id: -1,
+                    id: getRandomId(),
```

Comment:
Can we use uuid() generators instead of random number?

Reply:
No because the type of id is number and not string